src/SFA.DAS.AdminService.Web/RouteNames.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web
+{
+    public static class RouteNames
+    {
```

Comment:
Could you explain why you have these? They normally belong in the Controller and the HtmlHelper, TagHelper, etc will pick them up

Reply:
By using a const it makes refactoring a lot easier should the routes change.  Its easier to see where dependencies are.  With magic strings to controller/action its too easy to break links should an endpoint be moved.

Reply:
I'll let it go, normal convention tends to be magic strings or blank